#Device Service
# No Request needed
---
string Architecture
